src/test/java/org/folio/rest/impl/ProcessorTest.java

```diff
@@ -41,19 +47,31 @@
   private Requester requester;
 
   @Before
-  public void setUp() throws IOException {
+  public void setUp(TestContext ctx) throws IOException {
 
     MockitoAnnotations.initMocks(this);
     vertx = Vertx.vertx();
+    dummyResponse = createDummyResponse();
+    when(requester.post(anyString(), any(), anyMap())).thenReturn(dummyResponse);
+
     marcInputStream = this.getClass().getResourceAsStream("/msplit00000000.mrc");
+    twoMarcInstances = this.getClass().getResourceAsStream("/sourceRecords/msdb.bib.sub");
+
+    try (InputStream is = this.getClass().getResourceAsStream("/expected/msdb.bib.sub.out")) {
+      twoMarcInstancesOut = IOUtils.toString(is);
+    }
 
     Map<String, String> okapiHeaders = new HashMap<>();
-    processor = new Processor("testTenantId", okapiHeaders, requester, false);
-    dummyResponse = createDummyResponse();
+    processor = new Processor("testTenantId", okapiHeaders, requester, false, null);
 
     InputStream rules = this.getClass().getResourceAsStream("/rules.json");
     JsonObject rulesFile = new JsonObject(IOUtils.toString(rules));
     processor.setRulesFile(rulesFile);
+
+    processorSQLQueryTest = new Processor("testTenantId", okapiHeaders, requester, false,
+      "00000000-0000-0000-0000-000000000000");
```

Comment:
I changed it, but actually, the variable should NOT be called just `instanceId`, since it should only be set for this test purpose. It doesn't conflict with the patterns described by your links, I believe. Since the variable is always null, unless you set it in the constructor (which one should only do within a test). Like I said, it isn't a beautiful solution, but creating such tests afterwards without much of refactoring isn't easy. So, I'm thinking to rename it back to something like `testInstanceId`. I don't see why `testUuid` is wrong.